handlers/handler-loadbalance/src/test/java/org/apache/servicecomb/loadbalance/filter/TestIsolationServerListFilter.java

```diff
@@ -70,6 +75,14 @@ public void setUp() throws Exception {
     configuration.clearProperty("cse.loadbalance.isolation.enableRequestThreshold");
     configuration.addProperty("cse.loadbalance.isolation.enableRequestThreshold",
         "3");
+
+    taskList = new ArrayList<>();
+    EventManager.register(new Object() {
```

Comment:
global instance
it's better do not use this directly.

you can save a reference in filter
when UT create a new instance for it.